libs/dh/shared/assets/src/assets/b2c-custom-layouts/self-asserted.html

```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
suggestion: the changes are primarily formatting (except this one) but it's probably still a good idea to upload the file after merge. What do you think @frontend-specialisten :)